rootfs/scheduler/__init__.py

```diff
@@ -1053,25 +1047,39 @@ def get_secrets(self, namespace, **kwargs):
 
         return response
 
-    def create_secret(self, namespace, name, data, secret_type='Opaque', labels={}):
+    def _build_secret_manifest(self, namespace, name, data, secret_type='Opaque', labels=None):
```

Comment:
Since Python isn't strongly typed (or we aren't using it that way right now) then we anyway should be doing checks like that if we expect a dict. You can pass in anything as `labels` regardless of what I default it to